zcash_client_backend/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this library adheres to Rust's notion of
   - `ScannedBlock::sapling_tree_size`.
   - `ScannedBlock::orchard_tree_size`.
   - `wallet::propose_standard_transfer_to_address`
+  - `wallet::input_selection::Proposal::from_parts`
```

Comment:
This is `pub(crate)` so isn't part of the public API.
```suggestion
```

Reply:
Hm, perhaps it should be, though? Otherwise the only way to construct a proposal is via parsing, which is pretty inconvenient if you're trying to implement your own fee/change strategy.

Reply:
That seems fine, as long as `Proposal::from_parts` is enforcing all necessary internal consistency relations.